sys/include/net/gcoap.h

```diff
@@ -187,6 +190,106 @@
  * the Observe option value set to 1. The server does not support cancellation
  * via a reset (RST) response to a non-confirmable notification.
  *
+ * ## Block Operation ##
```

Comment:
Doxygen is generating this as a big blob, `###` are treated as comments and don't show up. Since this isn't a README
maybe consider using ` * =====` or ` * -------` do mark section limits.

Reply:
I don't follow you. When I generate the documentation, the web page output looks as I would expect. Can you be more specific? See the screenshot below.
.........

![pr11056-block-doc](https://user-images.githubusercontent.com/2944058/62990366-b912fd00-be19-11e9-8481-fc04e50e5759.png)

Reply:
That is indeed strange, when I generate it locally I get this layout:

![image](https://user-images.githubusercontent.com/23060007/63003731-2013ce80-be79-11e9-9cd3-a3208ed67c9b.png)

This is when calling `make doc` locally and opening the generated html. How are you opening/generating your file?

I see that in `http://riot-os.org/api` is the same you are showing. This looks like an issue with my doxygen version, I have `1.8.16` which should support markdown. Anyway if your version is rendering correctly then I trust the issue is on my side.

Reply:
I am using doxygen `1.8.13 ` from Ubuntu 19.04. I built the `1.8.16` release from source and regenerated the doc, with no change to the output. I display the doc from a local Apache web server. If I display from a `file:` URL, the fonts are different, but the structure is the same. So, I agree it seems that there is some difference in your setup.

I did notice that the level 3 headings for the block documentation in this PR do not include a closing `###` at the end of the line. The doxygen doc [shows](http://doxygen.nl/manual/markdown.html) that the absence of the closing hashes should not matter. However, I will add them to be consistent with the rest of the module documentation.